test/unit/spec/ForgotPassword_spec.js

```diff
@@ -18,7 +18,7 @@ import $sandbox from 'sandbox';
 const SharedUtil = internal.util.Util;
 const itp = Expect.itp;
 
-function setup(settings, startRouter) {
+async function setup(settings, startRouter) {
```

Comment:
Is this now a requirement for all Karma-based tests that use this approach for setup?

Reply:
no it's not a requirement, but I find it a little easier to debug and catch exceptions